elasticsearch-persistence/lib/elasticsearch/persistence/model.rb

```diff
@@ -121,8 +121,8 @@ def deserialize(document)
 
           # Set up common attributes
           #
-          attribute :created_at, DateTime, default: lambda { |o,a| Time.now.utc }
-          attribute :updated_at, DateTime, default: lambda { |o,a| Time.now.utc }
+          attribute :created_at, Time, default: lambda { |o,a| Time.now.utc }
+          attribute :updated_at, Time, default: lambda { |o,a| Time.now.utc }
```

Comment:
I could revert this.  I feel more strongly about ActiveSupport::TimeWithZone as in #395.  The rest of this PR is more important.